src/prefect/engine/cloud/flow_runner.py

```diff
@@ -155,6 +155,8 @@ def initialize_run(  # type: ignore
         updated_context = context or {}
         updated_context.update(flow_run_info.context or {})
         updated_context.update(
+            flow_id=flow_run_info.id,
+            flow_run_id=flow_run_info.id,
```

Comment:
Just to note: in the context of running w/ Cloud this is already added on to all deployments. (e.g. of use: https://github.com/PrefectHQ/prefect/blob/master/src/prefect/cli/execute.py#L39)